Discord Mafia Bot .Net/Commands/Commands.cs

```diff
@@ -39,6 +39,12 @@ public class MiscCommands : ModuleBase
             {
                 await Context.Message.AddReactionAsync(new Emoji("🍪"));
             }
+			
+			[Command("moo"), Summary("Moooooo!")]
```

Comment:
I told you, if you wanted to add this add a group called emoji's and put them at the bottom of the command class and then I'm fine with it. 😛